src/App.css

```diff
@@ -64,18 +65,18 @@ html, body{
 }
 .emoji-style{
   font-size: 25px;
-  height: 80%;
+  height: 70%;
   background-color: rgba(0,0,0,.25);
   border-radius: 50%;
   width: 25%;
   text-align: center;
 }
 .emoji-holder{
+  bottom: 104%;
```

Comment:
You shouldn't give that much percentage to bottom.Check another solution for this